### PR TITLE
feat(chat-e2e): unpublish conversation tests

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

unpublish conversation tests

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2640.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2640.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2640.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2640.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat-e2e/src/tests/publishConversation.test.ts [hunk 1]
```diff
@@ -415,9 +414,9 @@ dialAdminTest(
   }) => {
     setTestIds('EPMRTC-3575', 'EPMRTC-3584', 'EPMRTC-3589');
     const publicationNames = [
-      'name\t\twith\ttabs',
+      `${GeneratorUtil.randomPublicationRequestName()}name\t\twith\ttabs`,
       `  ${GeneratorUtil.randomPublicationRequestName()}  `,
-      'あおㅁㄹñ¿äß맞습니다. 한국어 학습의 인기는 그 나라의 문화와 경제뿐만 아니라 언어 자체의 매력에서도 비롯됩니다. 한국어는 한글이라는 고유한 문자 시스템을 사용하는데, 이는 15세기에 세종대왕에 의해 창안되었습니다. 한글은 그 논리적이고 과학적인 설계로 인해 배우기 쉬운 것으로 여겨지며, 이 또',
+      `${GeneratorUtil.randomPublicationRequestName()}あおㅁㄹñ¿äß맞습니다. 한국어 학습의 인기는 그 나라의 문화와 경제뿐만 아니라 언어 자체의 매력에서도 비롯됩니다. 한국어는 한글이라는 고유한 문자 시스템을 사용하는데, 이는 15세기에 세종대왕에 의해 창안되었습니다. 한글은 그 논리적이고 과학적인 설계로 인해 배우기 쉬운 것으로 여겨지며, 이 또`,
```
Comment (nartovm) [inline apps/chat-e2e/src/tests/publishConversation.test.ts]: I think that we need a constant for that set of symbols.

File: apps/chat-e2e/src/testData/api/publicationApiHelper.ts [hunk 2]
```diff
@@ -79,16 +79,17 @@ export class PublicationApiHelper extends BaseApiHelper {
...
+      });
+    }
     const data: PublicationRequestModel = {
       name: GeneratorUtil.randomUnpublishRequestName(),
       targetFolder: publicationRequest.targetFolder,
-      resources: resources,
+      resources: unpublishResources,
       rules: publicationRequest.rules,
     };
     const response = await this.request.post(API.publicationRequestCreate, {
```
Comment (nartovm) [inline apps/chat-e2e/src/testData/api/publicationApiHelper.ts:98]: I think that we need to add a response body validation, not only check fot 200 status

Comment (irinakartun) [reply apps/chat-e2e/src/testData/api/publicationApiHelper.ts:98]: fixed

File: apps/chat-e2e/src/tests/publishConversation.test.ts (hunk 1 above)
Comment (irinakartun) [reply apps/chat-e2e/src/tests/publishConversation.test.ts]: fixed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2640.nightly-test.deltixhub.io
>E2E tests status: success

**Review by nartovm — APPROVED** [review]

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**